project_type/models/project_type.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    task_type_id = fields.Many2one('project.type', 'Root type for Tasks')
+
+
+class ProjectType(models.Model):
+    _name = 'project.type'
```

Comment:
It's for the project.task. Please don't use the name: project.type. 

And actually there is already a class called: project.task.type which records the stage of the task. Please use the other name instead of the project.task.type.